prow/cluster/monitoring/mixins/grafana_dashboards/ghproxy-triage.jsonnet

```diff
@@ -0,0 +1,162 @@
+        uid: 'd72fe8d0400b2912e319b1e95d0ab1b3',
+    };
+
+dashboard.new(
+        'GitHub Cache',
+        time_from='now-7d',
+        schemaVersion=18,
+        refresh='1m',
+      )
+.addPanel(
+    (graphPanel.new(
+        'Cache Requests (per hour)',
+        description='Count of cache requests of each cache mode over the last hour.',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_avg=true,
+        legend_sort='avg',
+        legend_sortDesc=true,
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'sum(increase(ghcache_responses[1h])) by (mode)',
+        legendFormat='{{mode}}',
+    ))
+    .addTarget(prometheus.target(
+        'sum(increase(ghcache_responses{mode=~"COALESCED|REVALIDATED"}[1h]))',
+        legendFormat='(No Cost)',
+    )), gridPos={
+    h: 6,
+    w: 24,
+    x: 0,
+    y: 0,
+  })
+.addPanel(
+    (graphPanel.new(
+        'Cache Efficiency',
+        description='Percentage of cacheable requests that are fulfilled for free.\nNo cost modes are "COALESCED" and "REVALIDATED".\nCacheable modes include the no cost modes, "CHANGED" and "MISS".',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        legend_avg=true,
+        min='0',
+        max='1',
+        formatY1='percentunit',
+        #TODO: uncomment When this merges, https://github.com/grafana/grafonnet-lib/pull/122
+        #y_axis_label='% Cacheable Request Fulfilled for Free',
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'sum(increase(ghcache_responses{mode=~"COALESCED|REVALIDATED"}[1h])) \n/ sum(increase(ghcache_responses{mode=~"COALESCED|REVALIDATED|MISS|CHANGED"}[1h]))',
+        legendFormat='Efficiency',
+    )), gridPos={
+    h: 6,
+    w: 24,
+    x: 0,
+    y: 0,
+  })
+.addPanel(
+    (graphPanel.new(
+        'Disk Usage',
+        description='',
+        datasource='prometheus',
+        legend_alignAsTable=true,
+        legend_rightSide=true,
+        legend_values=true,
+        legend_current=true,
+        min='0',
+        max='1',
+        formatY1='percentunit',
+    ) + legendConfig)
+    .addTarget(prometheus.target(
+        'avg(ghcache_disk_used/(ghcache_disk_used+ghcache_disk_free)) without(instance)',
+        legendFormat='% Used',
+    ))
+    .addTarget(prometheus.target(
+        'avg(ghcache_disk_used) without(instance)',
+        legendFormat='GB Used',
+    ))
+    .addTarget(prometheus.target(
+        'avg(ghcache_disk_free) without(instance)',
+        legendFormat='GB Free',
+    ))
+    .addSeriesOverride({
+              alias: 'GB Used',
+              lines: false,
+              yaxis: 2,
+            })
+    .addSeriesOverride({
+              alias: 'GB Free',
+              lines: false,
+              yaxis: 2,
+            })
+    .resetYaxes()
```

Comment:
This does not look like it is copied from [the latest version](https://github.com/kubernetes/test-infra/blob/master/prow/cluster/monitoring/mixins/grafana_dashboards/ghproxy.jsonnet) of the ghproxy dashboard?
Need a rebase/refresh-copy?